benches/read_many_structs.rs

```diff
@@ -396,7 +396,7 @@ mod benchmark {
         let name = test_data_1_1.name.as_str();
 
         let empty_context = EncodingContext::for_ion_version(IonVersion::v1_1);
-        let compiled_macro = TemplateCompiler::compile_from_text(
+        let compiled_macro = TemplateCompiler::compile_from_source(
```

Comment:
🪧 The `compile_from_text` method previously required Ion 1.1. It now works with any Ion encoding; when used during bootstrapping we only pass it Ion 1.0.